Frameworks/UIKit/UIFontDescriptor.mm

```diff
@@ -51,7 +51,7 @@
 NSString* const UIFontFeatureTypeIdentifierKey = @"UIFontFeatureTypeIdentifierKey";
 NSString* const UIFontFeatureSelectorIdentifierKey = @"UIFontFeatureSelectorIdentifierKey";
 
-NSString* const SystemFontName = @"SegoeUI";
```

Comment:
I think Segoe UI was the only font we supported, so we hardcoded it in to work at some lower level when interfacing with FreeType? In any case, DWrite tells us the names have spaces, so spaces it is.

---
In reply to: [78118727](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78118727) [](ancestors = 78118727)